frontend/src/metabase/api/bug-report.ts

```diff
@@ -18,7 +18,14 @@ export const bugReportApi = Api.injectEndpoints({
         body,
       }),
     }),
+    getBugReportEnabled: builder.query<{ enabled: boolean }, void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/util/bug_report_enabled",
+      }),
+    }),
```

Comment:
you should not need another api endpoint to get a setting. you should be able to use

```ts
const isBugReportingEnabled = useSetting('bug-reporting-enabled');
```